data/kernels/color_conversion.cl

```diff
@@ -0,0 +1,185 @@
+/*
+    This file is part of darktable,
+    copyright (c) 2019 edgardo hoszowski.
+
+    darktable is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    darktable is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "common.h"
+
+// must be in synch with dt_iop_colorspace_type_t in imageop.h
```

Comment:
I suppose that ideally we'd want to move this definition into imageop_common.h and include this here and in imageop.c.

Reply:
Yes, all color things should be somewhere else, but I'm not sure exactly where, we can discuss it.